apps/site/src/content/blog/state-of-the-budgie-2023.mdx

```diff
@@ -0,0 +1,201 @@
+
+Budgie 10.8 was the first release to start requiring Magpie v0.x and the result has been a more consistent and reliable experience for our users.
+
+## Wayland efforts
+
+While our goal in 2023 was only to start experimenting with Wayland compositors and to take the learnings from those experiments to apply to a "future Budgie WM compositor" built for Budgie 11, we made a significant shift in our development focus starting middle of last year to accelerate our Wayland plans and work towards Budgie 10 being Wayland-only (and continuing to be Wayland-only going into Budgie 11). This was in recognition of several major developments in the broader ecosystem that we covered in our [Wayland](/wayland) post.
+
+In that lengthy post, we covered our experiences at that point with wlroots as well as XFCE's libxfce4windowing library, which provides us a library which supports both X11 (via libwnck) and Wayland (various Wayland protocols) at the same time. This is especially useful as we iterate on making Budgie Wayland-compatible while using it at the same time.
+
+I am happy to report that progress on Wayland support has been steady on all fronts since our shift in development focus to it.
+
+In Budgie Desktop, several applets and UX components of Budgie Desktop have been ported over or are in the process of being ported:
+
+- Show Desktop Applet has been ported
+- Tab Switcher has been ported
+- Workspaces Applet has been ported
+- IconTasklist Applet is currently being ported.
+
+Keep in mind that not every part of the codebase needs to be touched, but some key areas in the codebase will need to be updated once some newer window tracking logic has been merged in from the IconTasklist conversion: MenuManager (right-click menus), Panel Manager (manages panels), the legacy Tasklist needs to be rewritten, and various uses of Gdk.X11 APIs.
+
+On the compositor front, the progress by Campbell Jones (aka serebit) has been fantastic as well. Budgie's Wayland compositor, codenamed Magpie v1, was started in the form of a separate project called Flyaway at the end of April 2023. Since then, Magpie v1 has been fleshed out into a reasonably functional window manager with support for things X11 could only dream of. It's still a ways from becoming ready to integrate into Budgie proper, but you can already use it to run Waydroid and first-person Steam games (though variable refresh rate and direct scanout support are still pending)!
```

Comment:
Am curious as to why only first-person is specified. How the game does POV shouldn't matter, right?

Reply:
Yea I'm just gonna change it to "games through Steam"